### PR TITLE
[bn] Localized the content/bn/event-driven-architecture.md

### PR DESCRIPTION
**Pull request by ai-naymul**
### Describe your changes
Localize the content/bn/event-driven-architecture.md into bengali

### Related issue number or link (ex: `resolves #issue-number`)
Tracking No: #410 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/bn/event-driven-architecture.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: ঘটনা-চালিত স্থাপত্য
```
Comment (mitul3737) [inline content/bn/event-driven-architecture.md:2]:
```suggestion
title: ঘটনা-চালিত স্থাপত্য (Event-Driven Architecture)
```

File: content/bn/event-driven-architecture.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+category: ধারণা
+tags: ["architecture", "", ""]
+---
+
+## এটা কি
+
+ঘটনা চালিত স্থাপত্য হল একটি সফ্টওয়্যার স্থাপত্য যা ঘটনা তৈরি, প্রক্রিয়াকরণ এবং ব্যবহারকে প্রচার করে।
+একটি ঘটনা হল একটি অ্যাপ্লিকেশনের অবস্থার পরিবর্তন।
+উদাহরণস্বরূপ, একটি রাইড-শেয়ারিং অ্যাপে রাইডের প্রশংসা করা একটি ঘটনার প্রতিনিধিত্ব করে৷
+এই স্থাপত্যটি এমন কাঠামো তৈরি করে যেখানে ঘটনাগুলো তাদের উত্স থেকে (অ্যাপটি একটি রাইডের অনুরোধ করে) থেকে পছন্দসই রিসিভারগুলিতে (আশেপাশে উপলব্ধ ড্রাইভারদের অ্যাপগুলি) সঠিকভাবে রুট করা যেতে পারে।
```
Comment (mitul3737) [inline content/bn/event-driven-architecture.md:13]:
```suggestion
এই স্থাপত্যটি এমন কাঠামো তৈরি করে যেখানে ঘটনাগুলো তাদের উৎস থেকে (অ্যাপটি একটি রাইডের অনুরোধ করে) থেকে পছন্দসই রিসিভারগুলিতে (আশেপাশে উপলব্ধ ড্রাইভারদের অ্যাপগুলি) সঠিকভাবে রুট করা যেতে পারে।
```

File: content/bn/event-driven-architecture.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+যত বেশি তথ্য বাস্তব সময় পরিণত হয়, ঘটনাগুলো ক্যাপচার করা এবং উপযুক্ত [সার্ভিস](/service/) যা ঘটনার অনুরোধগুলি প্রক্রিয়া করতে হবে তা নিশ্চিত করার জন্য নির্ভরযোগ্য উপায়গুলি খুঁজে পাওয়া ক্রমশ চ্যালেঞ্জিং হয়ে উঠছে৷
```
Comment (mitul3737) [inline content/bn/event-driven-architecture.md:17]: Kindly make this a more suitable sentence "যত বেশি তথ্য বাস্তব সময় পরিণত হয়, ঘটনাগুলো ক্যাপচার করা এবং উপযুক্ত [সার্ভিস](/bn/service/) যা ঘটনার অনুরোধগুলি প্রক্রিয়া করতে হবে তা নিশ্চিত করার জন্য নির্ভরযোগ্য উপায়গুলি খুঁজে পাওয়া ক্রমশ চ্যালেঞ্জিং হয়ে উঠছে৷"

Comment (mitul3737) [inline content/bn/event-driven-architecture.md:17]:
```suggestion
যত বেশি তথ্য বাস্তব সময় পরিণত হয়, ঘটনাগুলো ক্যাপচার করা এবং উপযুক্ত [সার্ভিস](/bn/service/) যা ঘটনার অনুরোধগুলি প্রক্রিয়া করতে হবে তা নিশ্চিত করার জন্য নির্ভরযোগ্য উপায়গুলি খুঁজে পাওয়া ক্রমশ চ্যালেঞ্জিং হয়ে উঠছে৷
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by mitul3737** [comment]
@ai-naymul  You have done a very great job  Although I have asked for some changes here. Feel free to make those changes.

**Comment by ai-naymul** [comment]
> 

Thank You let me fix those :)

**Review by mitul3737 — APPROVED** [review]

**Review by Imtiaz1234 — APPROVED** [review]

**Comment by mitul3737** [comment]
Great job @ai-naymul ! Glad that you contributed to this project. I hope to see you contribute more in this project.

**Comment by ai-naymul** [comment]
> 

Thanks a lot @mitul3737 bhaiya for giving me this opportunity and ofcourse I am also Looking forward to contribute more on this 😊

**Pull request merged**